pkg/branch/setupProw.go

```diff
@@ -33,6 +33,13 @@ func SetupProw(manifest model.Manifest, release string, dryrun bool) error {
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("failed to generate new prow config: %v", err)
 	}
+
```

Comment:
Update the comment above to say commands since we are adding another one.

Reply:
Done.